src/components/TeamPlayer.js

```diff
@@ -0,0 +1,255 @@
+import React, { Component } from 'react';
+import {
+    View,
+    Text,
+    FlatList,
+    ImageBackground,
+    Image,
+    ScrollView,
+    StyleSheet} from 'react-native';
+import Team from '../models/team';
+import Player from '../models/player';
+
+const team = new Team();
+const player = new Player();
+
+const teams = [
+    {
+        id: team.id = 1,
```

Comment:
Move the data inside their own folder  and file

Reply:
You can move the teams content inside a file like this `src/data/teams.json`

Reply:
yeah its cool but There we are using team object ti create a team; and we can not initialize a class in a json file thats the main problem

Reply:
We will import the json data and pass them inside the variable

Reply:
But in that case you cant use classes to create those data

Reply:
@hervengenzi Did you do any research on how to load data from a JSON file ??

Reply:
Yeah i have done it

Reply:
From what I see the class objects you added within the object are useless. It's much easier to parse a JSON object into a class object than doing that. and it's much cleaner. Remember restful response always comes in as JSON and can always be converted to class objects

Reply:
Oh okay i got it